Zend/tests/generators/errors/yield_without_value_in_reference_generator_function.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
```

Comment:
I have added this extra test file but I am not sure if this needed since the `return_from_by_ref_generator.phpt` also tests the notice.